### DISCUSSION TITLE
Multiple Questions

### DISCUSSION THREAD

**Discussion by McFlyin619** [ditdot-dev/vue-flow-form#255] [Q&A] [answered]
I know this isn't what this tool is made for and defeats the purpose a bit, but i have a very long form and would like to add a few questions to display at the same time as the relate to each other.. is this possible or has anyone done it?

**Comment by EkaterinaVu** [comment, marked as answer]
Hi @McFlyin619, 

yes, it kind of defeats the purpose of displaying one question at a time which is our core concept. At the moment, it's not possible to display multiple questions at the same time and implementing this feature is not on our priority list. Still, we are aware that it is a highly requested feature so any contributions are welcome and greatly appreciated.
